node/cn/tracers/api.go

```diff
@@ -775,21 +758,23 @@ func (api *PrivateDebugAPI) traceTx(ctx context.Context, message blockchain.Mess
 			tracer = vm.NewInternalTxTracer()
 		} else {
 			// Constuct the JavaScript tracer to execute with
-			if tracer, err = tracers.New(*config.Tracer); err != nil {
+			if tracer, err = New(*config.Tracer); err != nil {
 				return nil, err
 			}
 		}
 		// Handle timeouts and RPC cancellations
 		deadlineCtx, cancel := context.WithTimeout(ctx, timeout)
 		go func() {
 			<-deadlineCtx.Done()
-			switch t := tracer.(type) {
-			case *tracers.Tracer:
-				t.Stop(errors.New("execution timeout"))
-			case *vm.InternalTxTracer:
-				t.Stop(errors.New("execution timeout"))
-			default:
-				logger.Warn("unknown tracer type", "type", reflect.TypeOf(t).String())
+			if deadlineCtx.Err() == context.DeadlineExceeded {
```

Comment:
Any reason using `==`? Ethereum uses `errors.Is`, right?

Reply:
Thanks for letting me know about this. Reflected at https://github.com/klaytn/klaytn/pull/1581/commits/397e334485b0c0bf5065cd6f63063e4d3313a0d4